src/common/rbac/const.go

```diff
@@ -77,3 +79,150 @@ const (
 	ResourceJobServiceMonitor  = Resource("jobservice-monitor")
 	ResourceSecurityHub        = Resource("security-hub")
 )
+
+var (
+	PoliciesMap = map[string][]*types.Policy{
```

Comment:
Because the item in this map has little difference with
https://github.com/goharbor/harbor/blob/738fde7d3bd8f201711907a826630b3fe5d13e16/src/common/rbac/project/rbac_role.go#L25
https://github.com/goharbor/harbor/blob/d4aa9b13c40343586bc70be08b66e2bb09ed84f6/src/common/rbac/system/policies.go#L23, 
could we add a new attribute NotReassignable to types.Policy?

Reply:
ditto